controllers/predicate.go

```diff
@@ -1,7 +1,7 @@
 package controllers
 
 import (
-	oadpApi "github.com/openshift/oadp-operator/api/v1alpha1"
+	oadpv1alpha1 "github.com/openshift/oadp-operator/api/v1alpha1"
```

Comment:
I think  this change may be outside the scope of the PR.

Reply:
Removed it from the PR!